tests/Behat/ContentLockContext.php

```diff
@@ -68,4 +116,42 @@ public function disableContentLock(): void {
     }
   }
 
+  /**
+   * Remove any created nodes.
+   *
+   * @AfterScenario
+   */
+  public function cleanNodes(): void {
+    if (\Drupal::moduleHandler()->moduleExists('content_lock')) {
+      \Drupal::database()->delete('content_lock')
```

Comment:
You are not cleaning nodes here, right? Then we should not indicate we do.

Reply:
I'm overriding the cleanNodes provided by the parent class though :(